CHANGELOG.md

```diff
@@ -14,6 +15,129 @@
+  `conformance/` directory.
+- The status of the GatewayClass "Accepted" condition for the `GatewayClass`
+  is now present in `kubectl get` output.
+  [#1168](https://github.com/kubernetes-sigs/gateway-api/pull/1168)
+- New `RouteConditionReason` types `RouteReasonNotAllowedByListeners` and
+  `RouteReasonNoMatchingListenerHostname` were added.
+  [#1155](https://github.com/kubernetes-sigs/gateway-api/pull/1155)
+- New `RouteConditionReason` type added with `RouteReasonAccepted`,
+  `RouteReasonResolvedRefs` and `RouteReasonRefNotPermitted` constants.
+  [#1114](https://github.com/kubernetes-sigs/gateway-api/pull/1114)
+- Routes can now select `Gateway` listeners by port number
+  [#1002](https://github.com/kubernetes-sigs/gateway-api/pull/1002)
+- Introduced PreciseHostname which prevents wildcard characters in relevant
+  Hostname values.
+  [#956](https://github.com/kubernetes-sigs/gateway-api/pull/956)
+
+[gep-917]:https://gateway-api.sigs.k8s.io/geps/gep-917/
+
+### Breaking Changes
+
+- The `NamedAddress` value for `Gateway`'s `spec.addresses[].type` field has
+  been deprecated, and support for domain-prefixed values (like
+  `example.com/NamedAddress`) has been added instead to better represent the
+  custom nature of this support.
+  [#1178](https://github.com/kubernetes-sigs/gateway-api/pull/1178)
+- `GatewayTLSConfig`'s `CertificateRefs` field is now a slice of pointers to
+  structs instead of the structs directly.
+  [#1176](https://github.com/kubernetes-sigs/gateway-api/pull/1176)
+- Implementations are now expected to use `404` instead of `503` responses when
+  the data-plane has no matching route.
+  [#1151](https://github.com/kubernetes-sigs/gateway-api/pull/1151)
+- The `UnsupportedExtension` named `ListenerConditionReason` has been removed.
+  [#1146](https://github.com/kubernetes-sigs/gateway-api/pull/1146)
+- The `RouteConflict` named `ListenerConditionReason` has been removed.
+  [#1145](https://github.com/kubernetes-sigs/gateway-api/pull/1145)
+- The `URLRewrite` filter `PreciseHostname` has replaced `Hostname`
+  [#1124](https://github.com/kubernetes-sigs/gateway-api/pull/1124)
+- `HTTPPathModifier` API replaces `Substitution` with new type-specific
+  `ReplaceFullPath` and `ReplacePrefixMatch` fields
+  [#1124](https://github.com/kubernetes-sigs/gateway-api/pull/1124)
+- Types `ConditionRouteAccepted` and `ConditionRouteResolvedRefs` are now
+  deprecated in favor of `RouteConditionAccepted` & `RouteConditionResolvedRefs`
+  [#1114](https://github.com/kubernetes-sigs/gateway-api/pull/1114)
+- URL Rewrites and Path redirects have been added as new "Experimental" features
+  [#945](https://github.com/kubernetes-sigs/gateway-api/pull/945)
+- The v1alpha1 API version has been deprecated.
+  [#906](https://github.com/kubernetes-sigs/gateway-api/issues/906)
+
+#### Validation Improvements
+
+The following are **breaking changes** where our webhook validation has
```

Comment:
I think we need to be very careful about labeling something as a breaking change. I would rather label these as "validation improvements" since they're preventing clearly invalid configuration.

Reply:
`Validation Improvements` are now they're own section and the informational text removed in 226a034767f9a076d3540f9dd98c2cd1627b385d